src/plot_api/plot_api.js

```diff
@@ -251,7 +251,7 @@ Plotly.plot = function(gd, data, layout, config) {
         // calc and autorange for errorbars
         ErrorBars.calc(gd);
 
-        // TODO: autosize extra for text markers
+        // TODO: autosize extra for text markers and images
```

Comment:
There's actually an issue for that one https://github.com/plotly/plotly.js/issues/1111